cmd/process-agent/api/server.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	settingshttp "github.com/DataDog/datadog-agent/pkg/config/settings/http"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/gorilla/mux"
+)
+
+func setupHandlers(r *mux.Router) {
+	r.HandleFunc("/config", settingshttp.Server.GetValue)
+	r.HandleFunc("/config", settingshttp.Server.GetFull("")).Methods("GET")
```

Comment:
What happens when you run this one?

Reply:
Oh!!!! This is actually a bug! Thank you so much for pointing that out to me